components/section/info/src/index.scss

```diff
@@ -0,0 +1,38 @@
+@import '~@schibstedspain/theme-basic/lib/index';
+@import './settings';
+
+.sui-SectionInfo {
+  @include media-breakpoint-down(m) {
+    flex-direction: column;
+  }
+  
+  border-bottom: 1px solid $c-sui-section-info-border;
```

Comment:
We should be EMMET compliant and declare some of our props as EMMET variables:
`$bdb-section-info: 1px solid $c-sui-section-info-border`

https://docs.emmet.io/cheat-sheet/